layer.md

```diff
@@ -61,6 +61,20 @@ Where supported, MUST include file attributes for Additions and Modifications in
 
 [Sparse files](https://en.wikipedia.org/wiki/Sparse_file) SHOULD NOT be used because they lack consistent support across tar implementations.
 
+#### Implied Directories
+
+As the tar format describes directory hierarchies using a flat datastructure, it is possible to have so-called "implied directories" where not all parent directories implied by an entries' path in the archive have their own entry.
+
+When applying a layer, implementations MUST create any parent directories implied by an entries' path, even if it is otherwise absent from the archive. Attributes of the created parent directories MUST be set as follows:
```

Comment:
I think this first MUST is uncontroversial, but the second MUST is potentially sticky and perhaps needs to be SHOULD?

To state that another way, are we confident that all existing implementations are currently complying with this second MUST? (I'm reasonably confident they are complying with the first one, because it's kind of unavoidable.)

Reply:
Friendly ping @neersighted :bow:

Reply:
I agree that the attribute list needs to be a SHOULD, if only because requiring empty `xattrs` can run into issues (for instance, SELinux labels and NFSv4 xattrs both have weird behaviours in this respect).